### DISCUSSION TITLE
Homework 1 Question 2.11

### DISCUSSION THREAD

**Discussion by edoyle724** [cooperrc/engineering-dynamics#9] [General]
I'm getting tripped up with the aerodynamic drag portion of this problem. I'm not sure if it's even necessary to answer the question? I think it is possible to answer this with just Newton's Second Law alone, but can you provide some guidance? Thanks!

**Comment by EthanOyo** [comment]
You can for sure use the second law to explain your answer. The skiers are being impacted by gravity, friction, and drag. Since a=F/m, you divide all your forces by the mass and you should find that the mass cancels for some of the forces, but not all. This should help lead you on the right track.
